yb-voyager/cmd/importData.go

```diff
@@ -1055,19 +1061,34 @@ func getYBSessionInitScript() []string {
 	fileScanner := bufio.NewScanner(varsFile)
 
 	var curLine string
+	var sessionVars []string
 	for fileScanner.Scan() {
 		curLine = strings.TrimSpace(fileScanner.Text())
-		sessionVars = append(sessionVars, curLine)
+		if checkSessionVariableSupport(curLine) {
```

Comment:
Add a check to confirm that the `curLine` is not empty.

Reply:
ok